### PR TITLE
AWS Route53 - add secret key and access key + AWS api module

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [https://github.com/demisto/etc/issues/36450](https://github.com/demisto/etc/issues/36450)

## Description
1. All our aws should support access with access key and secret key
2. GuardDuty contains a bug with the limit value

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by tallieber** [comment]
@guykeller maybe we will need to force merge since it does not have test playbook. This is the last aws integration that did not give the option to configure instance with secrets credentials.

File: Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.py [hunk 1]
```diff
@@ -338,29 +290,62 @@ def test_function():
...
+        if demisto.command() == 'aws-route53-waiter-resource-record-sets-changed':
+            result = waiter_resource_record_sets_changed(aws_client, args)
+
+        if demisto.command() == 'aws-route53-test-dns-answer':
+            result = test_dns_answer(aws_client, args)
+
+        demisto.results(result)
+        sys.exit(0)
+    except Exception as e:
+        return_error('Error has occurred in the AWS GuardDuty Integration: {error}\n {message}'.format(
```
Comment (guykeller) [inline Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.py:343]: please change GuardDuty to Route53

File: Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.yml [hunk 2]
```diff
@@ -16,11 +16,65 @@ configuration:
...
+  defaultvalue: 5
+  additionalinfo: "The maximum number of retry attempts when connection or throttling errors
+  are encountered. Set to 0 to disable retries. The default value is 5 and the limit is 10.
+  Note: Increasing the number of retries will increase the execution time."
+  required: false
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
```
Comment (guykeller) [inline Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.yml:73]: If a proxy can somehow be used (for example by boto3) let's use handle proxy.
If for some reason this integration cannot, under any circumstances, be used with a proxy - let's remove this parameter

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.yml (hunk 2 above)
Comment (tallieber) [reply Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53.yml:73]: AWS Api module is using the proxy with handle_proxy

**Review by guykeller — APPROVED** [review]

File: Packs/AWS-Route53/ReleaseNotes/1_2_0.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### AWS - Route53
+- Added the *Access Key* and *Secret Key* parameters to enable authentication using without a *roleArn*.
```
Comment (guykeller) [inline Packs/AWS-Route53/ReleaseNotes/1_2_0.md:3]:
```suggestion
- Added the *Access Key* and *Secret Key* parameters to enable authentication without using a *roleArn*.
```

File: Packs/AWS-Route53/ReleaseNotes/1_2_0.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+- Added the Timeout parameter, which enables you to define the amount of time until a timeout exception is reached.
```
Comment (guykeller) [inline Packs/AWS-Route53/ReleaseNotes/1_2_0.md:4]:
```suggestion
- Added the *Timeout* parameter, which enables you to define the amount of time until a timeout exception is reached.
```

**Review by guykeller — CHANGES_REQUESTED** [review]

File: Packs/AWS-Route53/ReleaseNotes/1_2_0.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### AWS - Route53
+- Added the *Access Key* and *Secret Key* parameters to enable authentication without using a *roleArn*.
+- Added the *Timeout* parameter, which enables you to define the amount of time until a timeout exception is reached.
+- Added the Retries parameter, which enables you to define the maximum retry attempts when connection or throttling errors
+    are encountered.
```
Comment (guykeller) [inline Packs/AWS-Route53/ReleaseNotes/1_2_0.md:6]:
```suggestion
- Added the Retries parameter, which enables you to define the maximum retry attempts when connection or throttling errors are encountered.
```

**Review by guykeller — APPROVED** [review]

**Pull request closed without merging**